### PR TITLE
Support llama3.2 LLM models in turbomind engine

### PR DESCRIPTION
**Pull request by lvhan028**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ For detailed inference benchmarks in more devices and more settings, please refe
   <li>Llama2 (7B - 70B)</li>
   <li>Llama3 (8B, 70B)</li>
   <li>Llama3.1 (8B, 70B)</li>
+  <li>Llama3.1 (1B, 3B)</li>
```
Comment (fanqiNO1) [inline README.md:118]: Typo.

Comment (lvhan028) [reply README.md]: fixed

File: docs/zh_cn/quantization/w4a16.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ lmdeploy serve gradio ./internlm2_5-7b-chat-4bit --server-name {ip_addr} --serve
 
 ## 模型评测
 
-我们使用 [OpenCompass](https://opencompass.readthedocs.io/zh-cn/latest/index.html) 评测量化模型在各个维度上的能力
+我们使用 [OpenCompass](https://opencompass.readthedocs.io/zh-cn/latest/index.html) 评测量化模型在各个维度上的能力。方法请参考[此处](https://opencompass.readthedocs.io/zh-cn/latest/advanced_guides/evaluation_lmdeploy.html)
```
Comment (AllentDan) [inline docs/zh_cn/quantization/w4a16.md:75]: Shall we update en doc as well?

**Review by AllentDan — APPROVED** [review]
Tested Llama-3.2-3B-Instruct OK for turbomind and pytorch both.

File: docs/zh_cn/quantization/w4a16.md (hunk 2 above)
Comment (lvhan028) [reply docs/zh_cn/quantization/w4a16.md:75]: Sure. Updated.

**Review by irexyc — APPROVED** [review]

**Comment by zhyncs** [comment]
@lvhan028 @AllentDan @irexyc Hi all Do we support Llama 3.2 Vision models right now

**Comment by zhyncs** [comment]
Just discovered that I missed Llama3.2-vision (11B, 90B) in the document, it seems to be supported now, thank you very much!

**Comment by zhyncs** [comment]
BTW I did a simple test and found that currently only the PyTorch engine supports Qwen 2 VL and Llama 3.2 Vision Instruct 11B. I used the following command in the latest docker image and found that the performance is not good.
```
curl -X POST http://localhost:8000/v1/chat/completions \
-H "Content-Type: application/json" \
-d '{
  "model": "meta-llama/Llama-3.2-11B-Vision-Instruct",
  "messages": [
    {
      "role": "user",
      "content": [
        {
          "type": "text",
          "text": "Describe this image in one sentence."
        },
        {
          "type": "image_url",
          "image_url": {
            "url": "https://picsum.photos/id/237/200/300"
          }
        }
      ]
    }
  ],
  "stream": true,
  "max_tokens": 512,
  "temperature": 0.5
}'
```

**Comment by lvhan028** [comment]
This PR is about the llama3.2 LLM model rather than the vision model
Sorry about the unclear PR title
```
| Llama3.2 | 3B | LLM | Yes | Yes | Yes | Yes |
```

**Pull request merged**